doc/src/base/base.md

```diff
@@ -228,6 +228,7 @@ Base.missing
 Base.coalesce
 Base.ismissing
 Base.skipmissing
+Base.something
```

Comment:
Should be listed with `nothing` and `Some` instead, it's totally unrelated to `missing`.